modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -2,162 +2,160 @@
 
 = Setting up {org-name} {product-name} on OpenShift
 
-To use {product-name}, you must run OpenShift and install an add-on which enables {product-name}. For more information about OpenShift, see link:https://www.openshift.org/[OpenShift website, window="_blank"].
-
-NOTE: {org-name} recommends running OpenShift using the method described in this document. However, there are many ways to run OpenShift, and {product-name} is compatible with most of these methods. There is a known issue with Minishift that makes it unsuitable for mobile development. This issue is described in link:https://github.com/minishift/minishift/issues/1287[Issue 1287, window="_blank"].
+To use {product-name}, you must have OpenShift instance and run installation script which enables {product-name}. For more information about OpenShift, see link:https://www.openshift.org/[OpenShift website, window="_blank"].
 
 [[prerequisites]]
 == Prerequisites
 
-* MacOS or Linux
```

Comment:
why are we removing this?

Reply:
This was for community release which was basically local OpenShift with mobile.next. For this mac/linux was needed. With tech preview you don't need it. You just need OpenShift instance.

Reply:
@jhellar @laurafitzgerald  I see the mdc-docs branch as being about landing new console. The work associated with 'tech preview' is different, and should land separately?

Reply:
@finp agreed the changes here should relate to changes generated by move to mdc

Reply:
@laurafitzgerald @finp probably should not have used term tech preview. But this is change related to move to mdc.